docs/service-loadbalancer.md

```diff
@@ -230,22 +230,25 @@ support BGP.
 
 MetalLB will automatically allocate external IPs for every Service of type
 LoadBalancer, and it sets the allocated IP to the `loadBalancer.ingress` field
-in the Service resource `status`. MetalLB also supports user specified `loadBalancerIP`
-in the Service spec. For more information, please refer to the [MetalLB usage](https://metallb.universe.tf/usage).
+in the Service resource `status`. MetalLB supports `spec.loadBalancerIP` and a custom `metallb.universe.tf/loadBalancerIPs`
+annotation in the Service spec.
```

Comment:
As annotation is not part of spec but metadata, I think it could be either of the followings:
* MetalLB supports `spec.loadBalancerIP` and a custom `metallb.universe.tf/loadBalancerIPs` annotation.
* MetalLB supports `loadBalancerIP` in the Service spec and a custom `metallb.universe.tf/loadBalancerIPs` annotation.